crates/wash-cli/src/dev.rs

```diff
@@ -1527,22 +1534,35 @@ async fn run_dev_loop(
         emoji::CONSTRUCTION_BARRIER,
         style("Building project...").bold(),
     );
-    let artifact_path = build_project(project_cfg, Some(&SignConfig::default()))
+    let Ok(Ok(artifact_path)) = build_project(project_cfg, Some(&SignConfig::default()))
         .await
-        .context("failed to build project")?
-        .canonicalize()
-        .context("failed to canonicalize path")?;
+        .context("failed to build project")
+        .map(|p| p.canonicalize().context("failed to canonicalize path"))
```

Comment:
I *wonder* if it's worth putting *which* path in here.. Not sure the canonicalize error includes the path it was actually trying to canonicalize